raiden_contracts/tests/test_channel_unlock.py

```diff
@@ -598,6 +617,7 @@ def test_channel_unlock(
         reveal_secrets,
         event_handler,
 ):
+    """ unlock() on pending transfers should split the locked amount into both parties """
```

Comment:
`unlock() on pending transfers with unlockable and expired locks should split the locked amount accordingly, to both parties`